go/runtime/txpool/txpool.go

```diff
@@ -28,6 +29,12 @@ const (
 	checkTxTimeout = 15 * time.Second
 	// checkTxRetryDelay is the time to wait before queuing a check tx retry.
 	checkTxRetryDelay = 1 * time.Second
+	// checkTxWaitForNodeSyncRetryDelay is the time to wait before retrying
+	// check tx while waiting for the node to sync.
+	checkTxWaitForNodeSyncRetryDelay = 30 * time.Second
+	// checkTxWaitRoundSyncedTimeout is the time to wait for block to be
+	// synced with storage at the start of check tx.
+	checkTxWaitRoundSyncedTimeout = 15 * time.Second
```

Comment:
I would reduce this timeout to something like 5 seconds as it will be retried anyway.

Reply:
Done :)  I've removed `checkTxWaitForNodeSyncRetryDelay` as suggested by Peter and reduced `checkTxWaitRoundSyncedTimeout` to 5 seconds.